### ISSUE TITLE
Problem with Pastebin module in (arch) linux

### ISSUE THREAD

**Issue by Nibodhika** [scottrice/Ice#354]
I'm getting:
```
ImportError: No module named Pastebin
```
After the normal setup.
Apparently according to [this](https://www.pythonanywhere.com/forums/topic/2903/) the solution is to install version 1.1.0, so after doing
```
pip2.7 install --user pastebin==1.1.0 --upgrade
```
and changing
```
import Pastebin as pastebin
```
to
```
import pastebin
```
I can launch ice.

**Comment by scottrice** [comment]
Thanks for reporting! Thats really odd behavior by the pastebin module. I have a fix incoming.

**Comment by scottrice** [comment]
Fixed by 4152a88ed52bffb6ad34d0386bae8e5aa313bea4
